extensions/amp-lightbox/1.0/lightbox.js

```diff
@@ -0,0 +1,239 @@
+  const animateInRef = useRef();
+  animateInRef.current = animateIn;
+  const lightboxContainWrapperRef = useRef();
+  lightboxContainWrapperRef.current = `.${styleClasses.lightboxContainWrapper}`;
+
+  useImperativeHandle(
+    ref,
+    () => ({
+      open: () => {
+        beforeOpen();
+        setShow(true);
+      },
+      close: () => {
+        setShow(false);
+      },
+    }),
+    [beforeOpen]
+  );
+
+  useEffect(() => {
+    if (show) {
+      setVisible(true);
+    } else {
+      getRefElement(lightboxRef).then((element) => {
+        element = element.current;
+        if (element != undefined) {
+          const animation = element.animate(
+            ANIMATION_PRESETS[animateInRef.current],
+            {
+              duration: ANIMATION_DURATION,
+              direction: 'reverse',
+              fill: 'both',
+              easing: 'ease-in',
+            }
+          );
+          animation.onfinish = (opt_anim) => {
+            setVisible(false);
+            setStyle(
+              opt_anim.currentTarget.effect.target,
+              'opacity',
+              ANIMATION_PRESETS[animateInRef.current][0].opacity
+            );
+            setStyle(
+              opt_anim.currentTarget.effect.target,
+              'visibility',
+              ANIMATION_PRESETS[animateInRef.current][0].visibility
+            );
+            afterClose();
+          };
+          return () => {
+            animation.cancel();
+            animation.onfinish();
+            afterClose();
+          };
+        }
+      });
+    }
+  }, [show, afterClose]);
+
+  useEffect(() => {
+    if (!show && !visible) {
+      return;
+    }
+    if (visible) {
+      getRefElement(lightboxRef).then((element) => {
+        element = element.current;
+        const animation = element.animate(
+          ANIMATION_PRESETS[animateInRef.current],
+          {
+            duration: ANIMATION_DURATION,
+            fill: 'both',
+            easing: 'ease-in',
+          }
+        );
+        animation.onfinish = (opt_anim) => {
+          setStyle(
+            opt_anim.currentTarget.effect.target,
+            'opacity',
+            ANIMATION_PRESETS[animateInRef.current][1].opacity
+          );
+          setStyle(
+            opt_anim.currentTarget.effect.target,
+            'visibility',
+            ANIMATION_PRESETS[animateInRef.current][1].visibility
+          );
+          opt_anim.currentTarget.effect.target.focus();
+        };
+        return () => {
+          animation.cancel();
+          animation.onfinish();
+        };
+      });
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [visible]);
+
+  return (
+    visible && (
+      <ContainWrapper
+        id={id}
+        ref={(r) => {
+          lightboxRef.current = r;
+        }}
+        style={{visibility: 'hidden'}}
+        size={true}
+        layout={true}
+        paint={true}
+        class={`${styleClasses.lightboxContainWrapper}`}
```

Comment:
`className`. And also, it's just `className={styleClasses.lightboxContainWrapper}`.